packages/client/src/InvitedContributor.ts

```diff
@@ -0,0 +1,79 @@
+import {
+    AuthenticatedLocClient,
+    AddTokensRecordParams,
+    FetchParameters,
+    BlockchainSubmission,
+    withLocId
+} from "./LocClient.js";
+import { PublicLoc } from "./Public.js";
+import { SharedState } from "./SharedClient.js";
+import { LogionClient } from "./LogionClient.js";
+import { DateTime } from "luxon";
+import { requireDefined } from "./assertions.js";
+
+export class InvitedContributorApi {
+
+    constructor(args: {
+        sharedState: SharedState,
+        logionClient: LogionClient,
+
+    }) {
+        this.sharedState = args.sharedState;
+        this.logionClient = args.logionClient;
+    }
+
+    private readonly sharedState: SharedState;
+    private readonly logionClient: LogionClient;
+
+    async findContributedLocById(params: FetchParameters): Promise<InvitedContributorLoc | undefined> {
```

Comment:
Why not simply `findLocById`? The method is defined in the context of `InvitedContributorApi` anyway.

Reply:
Done.